lib/msal-browser/src/interaction_client/SilentAuthCodeClient.ts

```diff
@@ -113,18 +113,19 @@ export class SilentAuthCodeClient extends StandardInteractionClient {
             this.logger.verbose("Auth code client created");
 
             // Create silent handler
-            const silentHandler = new SilentHandler(
+            const interactionHandler = new InteractionHandler(
```

Comment:
Update the comment.
nit: Also isn't the name misleading a bit? `silent` should mean `no interaction` right? I know you did not introduce this now but just wondering if changing it is possible.

Reply:
What I'm slowly working towards is not having any "Handlers" at all anymore, so this will come in time.